yarn-project/noir-contracts.js/package.json

```diff
@@ -49,4 +49,4 @@
   "engines": {
     "node": ">=18"
   }
-}
\ No newline at end of file
+}
```

Comment:
Just run formatting in yarn-project and this happened. Master was not correctly formatted.